dsl-reference.md

```diff
@@ -234,6 +234,7 @@ The Serverless Workflow DSL defines a list of [tasks](#task) that **must be** su
 |:--|:---:|:---:|:---|
 | input | [`input`](#input) | `no` | An object used to customize the task's input and to document its schema, if any. |
 | output | [`output`](#output) | `no` | An object used to customize the task's output and to document its schema, if any. |
+| export | [`export`](#export) | `no` | An object used to customize the content of the workflow context. | 
```

Comment:
Shouldn't we add the schema, as requested by you and @JBBianchi ? It felt like you both wanted it.

Reply:
The schema is already there

Reply:
Sorry!